docker_cli/commands.py

```diff
@@ -149,6 +146,45 @@ def copy_dir(
     )
 
 
+def configure_cmake(
```

Comment:
I don't see any tests exercising this function yet.

Let's add a test where we run this step and then check that the build directory exists and contains a file named `CMakeCache.txt` in the resulting image. For bonus points, you could parse the `CMakeCache.txt` file and check that the values of `CMAKE_BUILD_TYPE` and `WITH_CUDA` match the expected values.

Reply:
Done, with bonus points.